libr/syscall/syscall.c

```diff
@@ -184,7 +184,7 @@ R_API RSyscallItem *r_syscall_item_new_from_string(const char *name, const char
 	}
 	o = strdup (s);
 	int cols = r_str_split (o, ',');
-	if (cols < 3) {
+	if (cols < 4) {
```

Comment:
<3 should be fine too

Reply:
No, because there is r_str_word_get0 (o, 3), which gets the 4th word. This is why there was an ASAN crash.